src/libs/tools/src/plugindatabase.cpp

```diff
@@ -379,6 +380,108 @@ std::vector<PluginSpec> ModulesPluginDatabase::lookupAllProvides (std::string co
 }
 
 
+class PluginVariantsDatabase::Impl
+{
+public:
+	Impl ()
+	{
+	}
+	~Impl ()
+	{
+	}
+	Modules modules;
```

Comment:
How can I add an additional member (`KeySet pluginconf`) in the inherited `Impl` class? Adding an inherited implementation of it for `PluginVariantDatabase` yields a redefinition error.